src/Tribe/iCal.php

```diff
@@ -74,7 +74,7 @@ public static function single_event_links() {
 		}
 
 		echo '<div class="tribe-events-cal-links">';
-		echo '<a class="tribe-events-gcal tribe-events-button" href="' . esc_url( tribe_get_gcal_link() ) . '" title="' . __( 'Add to Google Calendar', 'tribe-events-calendar' ) . '">+ ' . __( 'Google Calendar', 'tribe-events-calendar' ) . '</a>';
+		echo '<a class="tribe-events-gcal tribe-events-button" href="' . tribe_get_gcal_link() . '" title="' . __( 'Add to Google Calendar', 'tribe-events-calendar' ) . '">+ ' . __( 'Google Calendar', 'tribe-events-calendar' ) . '</a>';
```

Comment:
We could make a special escape function like we discussed on the call that could look somewhat like this:

```php
public function esc_gcal_url( $url ) {
  $url = str_replace( '%0a', 'TRIBE-GCAL-LINEBREAK', $url );
  $url = esc_url( $url );
  $url = str_replace( 'TRIBE-GCAL-LINEBREAK', '%0a', $url );
  return $url;
}
```

That should fully escape everything and then re-inject the desired `%0a`. Then we can escape on this line like so:

```php
href="' . $something->esc_gcal_url( tribe_get_gcal_link() ) . '"
```

Where does the escaping function belong? The googleCalendarLink method is in `Tribe__Events__Main` so it feels like it should be in the same location.  Really, those functions belong elsewhere but that's probably a larger conversation.